src/bin/tikv-ctl.rs

```diff
@@ -197,20 +197,26 @@ impl MvccDeserializable for Vec<u8> {
     }
 }
 
+#[derive(PartialEq, Debug)]
 pub struct MvccKv<T> {
     key: Key,
     value: T,
 }
 
 pub fn gen_mvcc_iter<T: MvccDeserializable>(db: &DB,
                                             key_prefix: &str,
+                                            prefix_encoded: bool,
```

Comment:
The `key` in arg changes to `key_prefix`, so I think `prefix_encoded` is more suitable.